### DISCUSSION TITLE
How i can upgrade dots and save my own custom config?

### DISCUSSION THREAD

**Discussion by wantdead** [kRHYME7/Hyde-cli#84] [Q&A] [answered]
When upgrading, I can choose which files to synchronize
But the question arises, with a large number of such files they will simply be overwritten and I have to manually add all my settings again, which is somewhat tedious

Perhaps there is some easier way?

**Comment by kRHYME7** [comment, marked as answer]
For a long period of time it's how Dotfiles work. We need to re structure everything and make a DE like approach but that's overkill for a Dotfile repo. I can suggest is to make a plugin system which has a meta file listing all the files to be transferred.That way we can still make it modular. This would still need a lot of decision making though 


Edit: Best way for every user is to make a Fork of this dotfiles. Pull any changes. resolve conflicts and merge.

**Reply by wantdead** [reply]
Hi, @kRHYME7 
thanks for the answer
Yes, I understand perfectly well what you write.
I was just hoping there was some way to solve the problem that I was missing.

I'll close the discussion
